graphql2/graphqlapp/alert.go

```diff
@@ -39,6 +41,36 @@ func (a *AlertLogEntry) Message(ctx context.Context, obj *alertlog.Entry) (strin
 	return e.String(), nil
 }
 
+func (a *AlertLogEntry) State(ctx context.Context, obj *alertlog.Entry) (*graphql2.AlertLogEntryState, error) {
+	e := *obj
+	meta, ok := e.Meta().(*alertlog.NotificationMetaData)
+	if !ok {
```

Comment:
Also, check that `meta` isn't nil -- as will be the case for invalid or missing metadata (e.g. old log entries made before this PR). Otherwise `meta.MessageID` will panic below.
```suggestion
	if !ok || meta == nil {
```